Dockerfile

```diff
@@ -66,20 +56,13 @@ RUN set -ex \
     && useradd -ms /bin/bash -d ${AIRFLOW_HOME} airflow \
     && pip install -U pip setuptools wheel \
     && pip install Cython \
-    && pip install kubernetes \
```

Comment:
kubernetes is needed for Kubernetes Executor, unless this is installed by airflow packages.